tools/ambreModelInterpolation4DRBF.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import pandas as pd
+from astropy.io import fits
+from astropy.table import Table
+from astropy.io import ascii
+from scipy.interpolate import Rbf
+import parallel
+
+
+class AmbreInterpolation: 
+    """ Interpolate a model spectrum along four model parameters using scipy.interpolate.Rbf.
+        `interpolate` method returns a `numpy.ndarray` of an interpolated spectrum.
+        This script calls `parallel.py`.
+
+    Parameters
+    ----------
+    teff : Effective temperature (Teff [K]) at an interpolation. type = float
+    logg : Log of surface gravity (g in units of cm/s2) at an interpolation. Type = float
+    metal : Mean stellar metallicity ([Metal/H] or [Fe/H]) at an interpolation. Type = float
+    alpha : [alpha/Fe] chemical index at an interpolation. Type = float
+    path2Spectrum : Path to a directory of spectra.
+    path2List : A list of models in the directory of spectra; the list have columns of 
+                [SepctrumName,Atmosphere,Teff,Logg,Mass,Turbv,Metal,Alpha]
+    function : Function for RBF interpolation; the default is `multiquadric`.
+    epsilon : Epsilon parameter in RBF (see scipy.interpolate.Rbf); the default is 2.0., Type = float
+    """ 
+
+    def __init__(self, teff, logg, metal, alpha, 
+                 path2Spectrum, path2List, 
+                 function='multiquadric', epsilon=2.0):
+        self.teff = teff
+        self.logg = logg
+        self.metal = metal
+        self.alpha = alpha
+        self.path2Spectrum = path2Spectrum
+        self.path2List = path2List
+        self.function = function
+        self.epsilon = epsilon
+
+    def modelList(self):
+        """ Read a list of spectra """
+        parameterList = ascii.read(self.path2List).to_pandas()
+        return parameterList
+    
+    def fetchReferenceModelName(self):
+        """ Fetch a reference model. If no reference, return an empty numpy array. """
+        parameterList = AmbreInterpolation.modelList(self)
+        referenceModelName = parameterList[(parameterList['Teff'] == self.teff) & 
+                                           (parameterList['Gravity'] == self.logg) & 
+                                           (parameterList['Metal'] == self.metal) & 
+                                           (parameterList['Alpha'] == self.alpha)]['ModelName'].values
+        if referenceModelName.size != 0:
+            return referenceModelName[0]
+        if referenceModelName.size == 0:
+            return np.array([])
+
+    def interpolate(self, nProcs=1):        
+        """ Do interpolation
+
+        Parameter
```

Comment:
Even though there is a single parameter here, the docString section should be 'Parameters'.